src/cm.css

```diff
@@ -699,8 +699,7 @@ a:focus-visible {
 }
 
 /** Scrollbar border */
-.scroller::-webkit-scrollbar,
-.scroller::-webkit-scrollbar-thumb,
+.scroller::-webkit-scrollbar, .scroller::-webkit-scrollbar-thumb,
```

Comment:
Did we have an issue with multiple selectors on one line? I remember you or Ben mentioning something like that

Reply:
no its fine on 1 line, it just cant be more than 2 lines of selectors